README.md

```diff
@@ -50,14 +50,14 @@ juju run {unit_name} get-redirect-uri --wait
 
 To disable provider, i.e remove it from Kratos, run:
 ```commandline
-juju run {unit_name} disable --wait
+juju config kratos-external-provider-integrator enabled=false
```

Comment:
isn't it `kratos-external-idp-integrator`?

Reply:
oh, nice catch

Fixed